gencred/BUILD.bazel

```diff
@@ -34,3 +35,12 @@ go_binary(
     embed = [":go_default_library"],
     visibility = ["//visibility:public"],
 )
+
+py_binary(
+    name = "merge_kubeconfig_secret",
+    srcs = ["merge_kubeconfig_secret.py"],
+    python_version = "PY3",
+    deps = [
+        requirement("google-cloud-secret-manager"),
```

Comment:
google-cloud-secret-manager depends on asteroid, thus you need to add `requirement('asteroid`)` in here (as well as any other dependencies

Reply:
Had an offline discussion, the error message with `asteroid` was probably a misleading error, which very likely was a truncated error message meant to print https://github.com/kubernetes/test-infra/blob/master/requirements3.txt but only got to the first line.